### PR TITLE
[DIA-2473] manage package

### PR DESCRIPTION
**Pull request by Nevazhnovu**

### CODE REVIEW

File: Assets/ConsentManagementProvider/Scripts/facade/CMP.cs [hunk 1]
```diff
@@ -11,8 +11,17 @@ namespace ConsentManagementProviderLib
...
+                                        || Application.platform == RuntimePlatform.WindowsEditor
+                                        || Application.platform == RuntimePlatform.OSXEditor;
+        
+        
+        public static void Initialize(
+            List<SpCampaign> spCampaigns, 
+            int accountId, string propertyName, 
+            MESSAGE_LANGUAGE language, 
+            CAMPAIGN_ENV campaignsEnvironment,
+            long messageTimeoutInSeconds = 3)
```
Comment (andresilveirah) [inline Assets/ConsentManagementProvider/Scripts/facade/CMP.cs:24]: Can you increase this to 30? We've had a few issues with timeout before.

File: Assets/ConsentManagementProvider/Scripts/facade/CMP.cs [hunk 2]
```diff
@@ -51,6 +60,12 @@ public static void Initialize(List<SpCampaign> spCampaigns, int accountId, strin
         
         public static void LoadMessage(string authId = null)
         {
+            if (IsEditor)
```
Comment (andresilveirah) [inline Assets/ConsentManagementProvider/Scripts/facade/CMP.cs:63]: What if we extract this logic to our own logger class? So you can always call the log method and inside the method you'd have that conditional `if (IsEditor)` .

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Unity-SDK
 Your plug &amp; play CMP for Unity.
 
-<mark>**Note**: Sourcepoint's Unity SDK can be used for both Android OS and iOS. When you are ready to compile your project for iOS, you will need to perform additional steps. [Click here](https://github.com/SourcePointUSA/unity-sdk#usage--build-for-ios) for more information.
+<mark>**Note**: Sourcepoint's Unity SDK can be used for both Android OS and iOS. It does nothing in Unity Editor due to architecture of SDK (it embeds native SDKs which can be accessed only when application is built for corresponding platform)
```
Comment (andresilveirah) [inline README.md:4]: I'd rephrase it like:

>  Sourcepoint's Unity SDK can be integrated with both Android and iOS. Since it embeds native SDKs, and those only work in their respective platforms, the Unity SDK can't be demoed using Unity's Editor.

File: README.md [hunk 4]
```diff
@@ -42,7 +43,7 @@ using ConsentManagementProviderLib;
 ```c#
 private void Start()
 {
-    CMP.LoadMessage(authId: "JohnDoe");
+    CMP.LoadMessage(authId: "JohnDoe"); // it may be null
```
Comment (andresilveirah) [inline README.md:46]:
```diff
-      CMP.LoadMessage(authId: "JohnDoe"); // it may be null
+     CMP.LoadMessage(authId: null); // or pass it a String if you wish to use authenticated consent
```

File: Assets/ConsentManagementProvider/Scripts/facade/CMP.cs (hunk 2 above)
Comment (Nevazhnovu) [reply Assets/ConsentManagementProvider/Scripts/facade/CMP.cs:63]: Warning message is a bit different in each and every call so I'd rather leave it as it is

Comment (andresilveirah) [reply Assets/ConsentManagementProvider/Scripts/facade/CMP.cs:63]: It's just a matter of receiving the message as a parameter :) 

```csharp
interface Logger
{
	void log(string message);
}

class EditorLogger: Logger
{
	public void log(string message) {}
}

class NonEditorLogger: Logger
{
	public void log(string message)
	{
        	Debug.LogWarning(message);
	}
}

// somewhere in the code
public Logger logger = isEditor ? new EditorLogger() : new NonEditorLogger();
```
